### PR TITLE
Update Image Dependencies

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This PR bumps the dependencies used for this Docker image.

## 💭 Motivation and context ##

Some of these packages are out-of-date and we were using an older Python image version.

## 🧪 Testing ##

Automated tests pass successfully. A local testing image was able to perform as expected.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
:shipit:

File: src/requirements.txt [hunk 1]
```diff
@@ -1,4 +0,0 @@
-docopt
-https://github.com/cisagov/hash-http-content/archive/v0.0.1.tar.gz
-requests
-urllib3
```
Comment (jsf9k) [inline src/requirements.txt:4]: Why is this file being deleted?  Doesn't `requirements-test.txt` still reference it?

**Review by jsf9k — COMMENTED** [review]
I have one important question.

File: src/requirements.txt (hunk 1 above)
Comment (mcdonnnj) [reply src/requirements.txt:4]: This is in the `src/` subdirectory.

Comment (dav3r) [reply src/requirements.txt:4]: `src/requirements.txt` is being deleted, but the [top-level `requirements.txt`](https://github.com/cisagov/vdp-scanner-docker/blob/maintenance/update_image_dependencies/requirements.txt) is still there.

Comment (jsf9k) [reply src/requirements.txt:4]: I am a fool.

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
LGTM! ⚡

**Pull request merged**